molo/core/models.py

```diff
@@ -444,10 +444,12 @@ class Tag(TranslatablePageMixin, Page):
     subpage_types = []
 
     feature_in_homepage = models.BooleanField(default=False)
+    feature_in_section = models.BooleanField(default=False)
```

Comment:
Do we still need this? If you're setting on a section level the tags a section should show, then this falls away, no?